qiskit_experiments/calibration_management/backend_calibrations.py

```diff
@@ -32,18 +30,10 @@
 from qiskit_experiments.calibration_management.basis_gate_library import BasisGateLibrary
 
 
-class FrequencyElement(Enum):
-    """An extendable enum for components that have a frequency."""
-
-    QUBIT = "Qubit"
-    READOUT = "Readout"
-
-
 class BackendCalibrations(Calibrations):
```

Comment:
Probably this is `IBMBackendCalibrations`? Assuming `control_config`, `qubit_freq`, `meas_freq` make the class too tightly coupled to IBM architecture. According to OpenPulse spec control channel is whatever can be defined as device Hamiltonian, so this may not keyed on tuple of qubits (so this should take callback function that returns control channel).

Reply:
This is still fine. For the control channels this corresponds to the current backend configuration in the provider models. See here: https://github.com/Qiskit/qiskit-terra/blob/e0d0f687d97ae3dd4563617cf48b1871930d05c5/qiskit/providers/models/backendconfiguration.py#L954-L966 The model `Tuple(qubits,...): List[ControlChannel]` can accommodate a wide range of architectures (tunable couplers, CZ-based, etc). It should also be able to accommodate non-superconducting qubit architectures as well since in the end we are assuming digital quantum computing which runs on circuits which apply to qubits. 

With 6509fdc we are robust to cases where the defaults do not have `qubit_freq_est` and `meas_freq_est`. Backends that define the qubit and readout frequency in another fashion can then add such parameters in another fashion.